### PR TITLE
Add arch migration

### PR DESCRIPTION
**Pull request by martin-g**
Based on top of https://github.com/conda-forge/perl-encode-feedstock/pull/4

* Uses Azure CI for PPC64LE (cross-compilation) because TravisCI often fails to provide a PPC64LE native node.
* Fix lint issues (`stdlib('c')`)

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by martin-g** [comment]
@conda-forge-admin, please rerender

**Comment by martin-g** [comment]
The PPC64LE build won't work because of:
```
Encountered problems while solving:
  - nothing provides requested perl-storable
```
I'll exclude it!

**Comment by martin-g** [comment]
@conda-forge-admin, please rerender

**Comment by martin-g** [comment]
@conda-forge-admin, please ping team
Please review!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,12 +11,13 @@ source:
...
 
 build:
-  number: 0
+  number: 1
   skip: true  # [win]
 
 requirements:
   build:
     - {{ compiler('c') }}
+    - {{ stdlib("c") }}
```
Comment (xileF1337) [inline recipe/meta.yaml]: Could you please use a consistent quotation style for these template directives (either double " " or single ' ', I'm fine with either)? Thank you!

**Review by xileF1337 — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (martin-g) [reply recipe/meta.yaml]: Done!

**Comment by martin-g** [comment]
@conda-forge-admin, please rerender

**Review by xileF1337 — APPROVED** [review]

**Pull request merged**